next.config.js

```diff
@@ -50,12 +50,23 @@ const nextConfig = {
     return [
       {
         // Apply these headers to all static assets
-        source: "/(.*)",
+        source: "/api/:path*",
```

Comment:
The last commit that changed this line says "Implement cache control headers for static assets".
Are you sure that access to static assets will not break because of this change?

I think those are needed for the Chrome extension itself.

Also, if you don't want to risk it, you can always create a new object in the returned array, no?